my-cloud-client.sublime-project

```diff
@@ -0,0 +1,33 @@
+{
+	"folders":
+	[
+		{
+			"file_exclude_patterns":
+			[
+				"*.gitignore"
+			],
+			"file_include_patterns":
+			[
+				"*.js",
+				"*.json",
+				"*.html",
+				"*.css"
+			],
+			"folder_exclude_patterns":
+			[
+				".git",
+				".vagrant",
+				"node_modules"
+			],
+			"name": "client",
+			"path": "client"
+		},
+		{
+			"name": "server",
+			"path": "server"
+		},
+		{
+			"path": "docs"
+		}
+	]
+}
```

Comment:
Do you like to have project file in the project ?
I mean, it is common practice to keep project files from different IDEs along with source code.

Although, .submilime-workspace should be removed from repo